bin/copy-labels.js

```diff
@@ -0,0 +1,288 @@
+  } catch (err) {
+    helpExit(`Error writing file: ${file}`, err);
+  }
+};
+
+const dryDeleteLabels = async (args, handler) => {
+  const labels = await handler.get();
+  const usedLabels = await handler.getUsedLabels(args.dstRepo);
+  console.log(`Delete following labels from '${args.dstRepo}'`);
+  printLabels(labels.filter(({ name }) => !usedLabels.has(name)));
+  return usedLabels;
+};
+
+const printAddedLabels = (args, labels, existingLabels, usedLabels) => {
+  console.log('Following labels will be added:');
+  printLabels(
+    labels.filter(label => {
+      for (const existingLabel of existingLabels) {
+        if (existingLabel.name === label.name) {
+          return args.delete && !usedLabels.has(label.name);
+        }
+      }
+      return true;
+    })
+  );
+};
+
+const printUpdatedLabels = (args, labels, existingLabels, usedLabels) => {
+  console.log('Following labels will be updated:');
+  console.log('from:');
+  printLabels(
+    labels.filter(label => {
+      for (const existingLabel of existingLabels) {
+        if (existingLabel.name === label.name) {
+          return !args.delete || usedLabels.has(label.name);
+        }
+      }
+      return false;
+    })
+  );
+  console.log('to:');
+  printLabels(
+    existingLabels.filter(existingLabel => {
+      for (const label of labels) {
+        if (existingLabel.name === label.name) {
+          return !args.delete || usedLabels.has(label.name);
+        }
+      }
+      return false;
+    })
+  );
+};
+
+const printUnchangedLabels = (args, labels, existingLabels, usedLabels) => {
+  console.log('Following labels will not be changed:');
+  printLabels(
+    existingLabels.filter(existingLabel => {
+      for (const label of labels) {
+        if (existingLabel.name === label.name) {
+          return !args.update && (!args.delete || usedLabels.has(label.name));
+        }
+      }
+      return !args.delete || usedLabels.has(existingLabel.name);
+    })
+  );
+};
+
+const dryCopyLabels = async (args, handler, labels, usedLabels) => {
+  if (args.dstFile) {
+    console.log(
+      `Write following labels ` +
+        `from repository '${args.srcRepo}' to file '${args.dstFile}'`
+    );
+    printLabels(labels);
+  } else {
+    const existingLabels = await handler.get();
+    let msg = 'Copy the following labels from';
+    if (args.srcFile) {
+      msg += ` file '${args.srcFile}'`;
+    } else {
+      msg += ` repository '${args.srcRepo}'`;
+    }
+    msg += ` to repository '${args.dstRepo}'`;
+    if (args.update) msg += ' updating existing labels';
+    console.log(msg);
+    printAddedLabels(args, labels, existingLabels, usedLabels);
+    if (args.update) {
+      printUpdatedLabels(args, labels, existingLabels, usedLabels);
+    }
+    printUnchangedLabels(args, labels, existingLabels, usedLabels);
+  }
+};
+
+const copyLabels = async (args, handler, labels) => {
+  if (args.dstFile) {
+    try {
+      await writeDump(args.dstFile, labels);
```

Comment:
You already have try-catch block in `writeDump()` function that leads to the exit, what's the point of adding one here as well?